library/src/scala/reflect/Typeable.scala

```diff
@@ -0,0 +1,12 @@
+package scala.reflect
+
+/** A shorhand for `TypeTest[Any, T]`. A `Typable[T] contains the logic needed to
+ *  know at runtime if a value can be downcasted to `T`.
+ *
+ *  If a pattern match is performed on a term of type `s: Any` that is uncheckable with `s.isInstanceOf[T]` and
+ *  the pattern are of the form:
+ *    - `t: T`
+ *    - `t @ X()` where the `X.unapply` has takes an argument of type `T`
+ *  then a given instance of `Typable[T]` (`TypeTest[Any, T]`) is summoned and used to perform the test.
+ */
+type Typable[T] = TypeTest[Any, T]
```

Comment:
I'm not a native English speaker, but shouldn't this be spelled `Typeable`?

Reply:
seems hackage has `Typeable`

Reply:
Indeed, it should be Typeable.